src/lib/gql/queries.ts

```diff
@@ -194,6 +197,85 @@ export const getDiscordWebhook = async (circleId: number) => {
   return circle_private.pop()?.discord_webhook;
 };
 
+export const getVaultAndTransactions = async (address: string | undefined) => {
```

Comment:
probably better to move this to its own file under `VaultsPage`. fits better with the pattern we've been establishing of queries being really fine-tuned to the needs of their calling pages, not to be reused in different contexts